Project4/index.html

```diff
@@ -0,0 +1,86 @@
+        your employer will be trying to wring every ounce of ROI out of your paycheck, even if it means having
+        you do a ton of non-design work. As a result, designers find themselves working outside the scope of their 
+        specialty and forced to learn new skills on the fly, including social media, website management, 
+        project management and even technical tasks.</p>
+        <p>Can specialists still make it in the creative industry? They can, but the demand for them is shrinking as 
+        more decision-makers look to cut costs, so it takes exceptional talent to do so.</p>
+        <p class="name"><b>LIZ GLANCY</b></p>
+        <a name="speakers"></a>
+        <img class="speaker" src="Glancy.jpg" alt="LGlancy">
+        <p><b>Speaking on June 20<small>th</small></b></p>
+        <p><b>REEBOK</br>
+        Title: Footwear Designer II</b></br>
+        Dates Employed: Jun 2003 – May 2011</br>
+        Employment Duration: 8 yrs</br>
+        Location: Canton, MA</p>
+               
+        <p><b>Graphic Designer for RBK Footwear</b></br>
+        Dates Employed: 2003 – 2005</br>
+        Employment Duration: 2 yrs</br>
+        Location: Canton, MA</p>
+        
+        <p class="name"><b>Ellen Lupton</b></p>
+        <img class="speaker" src="Lipton.jpg" alt="EllenLipton">
+        <p><b>Speaking on June 21<small>st</small></b></p>
+        <p><b>Author, Curator, and Graphic Designer</br> 
+        Director of the Graphic Design MFA a Maryland Institute College of Art (MICA)</b></p>
+        <p>Books include:</br> 
+        <i>Graphic Design The New Basics</i> (with Jennifer Cole Phillips, 2008)</br> 
+        <i>How to Design and Produce Your Own Book</i> (2008)</br> 
+        <i>Thinking with Type</i> (2004)</br> 
+        <i>D.I.Y.: Design It Yourself</i> (2006) co-authored with her graduate students at MICA.</p> 
+
+        <p class="name"><b>Jacob Lockard</b></p>
+        <img class="speaker" src="jlockard.jpg" alt="JacobLockard">
+        <p><b>Speaking on June 22<small>nd</small></b></p>
+        <p><b>Atlanta-based Graphic Designer</br> 
+        Senior Art Director at AKQA</p>
+        <p>Co-conspirator</b> of the Baltimore community drawing event known as Drawnk! (now defunct), 
+        which was held the first Tuesday of every month at The Windup Space in Baltimore, MD. 
+        It was a community arts  event that encouraged people to get together, draw, network and have a good time. 
+        Drawnk! conspirators were Rick Kercz, Justin Gilman & Josh Gilman. 
+        I'm always looking to start something new, so if you have a great idea and a reason to collaborate, 
+        I have a reason to join you.</p>
+
+        <a href="https://calendar.google.com/calendar/render?pli=1"><button class="button1">Calendar</button></a>
+        <a name="cart"></a>
+        <a href="https://www.expedia.com/event-tickets/"><button class="button1"><b>Buy Tickets</b></button></a>
+    </div class="site-wrapper">
```

Comment:
Classnames only need to be applied to opening tags:
```suggestion
    </div>
```